sql/5_0_0-to-5_0_1_upgrade.sql

```diff
@@ -661,3 +661,7 @@ DELETE FROM list_options WHERE list_id = 'lists' AND option_id = 'lbfnames';
 DELETE FROM list_options WHERE list_id = 'transactions';
 DELETE FROM list_options WHERE list_id = 'lists' AND option_id = 'transactions';
 #EndIf
+
+#IfRow2D layout_options source F edit_options %0%
```

Comment:
Above wildcard won't work. Let me think about this a bit.

Reply:
Just as a general comment, the sql update logic could be improved with new features as appropriate. One thing that would be nice is nesting of conditionals (#Ifxxx and #EndIf).

Reply:
It's tested and it's works but I agree that it's a bit strange.. do you prefer that I will add new function called 'IfRowLike'?

Reply:
If it works, then would keep it as is (assuming of course, it is even needed, if plan it to try to get the 0 to work).